src/gui/presets.cpp

```diff
@@ -54,65 +54,128 @@ void FurnaceGUI::initSystemPresets() {
 
   CATEGORY_BEGIN(_("Game consoles"),_("let's play some chiptune making games!"));
   ENTRY(
-    _("Sega Genesis"), {
-      CH(DIV_SYSTEM_YM2612, 1.0f, 0, ""),
-      CH(DIV_SYSTEM_SMS, 0.5f, 0, "")
-    }
+    _("Sega Genesis"), {}
   );
     SUB_ENTRY(
-      _("Sega Genesis (extended channel 3)"), {
-        CH(DIV_SYSTEM_YM2612_EXT, 1.0f, 0, ""),
-        CH(DIV_SYSTEM_SMS, 0.5f, 0, "")
-      }
-    );
-    SUB_ENTRY(
-      _("Sega Genesis (CSM)"), {
-        CH(DIV_SYSTEM_YM2612_CSM, 1.0f, 0, ""),
-        CH(DIV_SYSTEM_SMS, 0.5f, 0, "")
-      }
-    );
-    SUB_ENTRY(
-      _("Sega Genesis (DualPCM)"), {
-        CH(DIV_SYSTEM_YM2612_DUALPCM, 1.0f, 0, ""),
+      _("Sega Genesis Model 1"), {
```

Comment:
Reject. This will result in clutter when detecting system.
Revert this section.

Reply:
No.